.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build Docker Image
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Gitlab
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          platforms: linux/amd64,linux/arm64
+          tags: ${{ secrets.DOCKERHUB_PROJECT }}:latest
```

Comment:
I would recommend setting up tagging similar to https://github.com/artis3n/pgmodeler-container/blob/main/.github/workflows/deploy.yml#L31 such that, upon tagging a release in this repo, you get all the correct major, minor, patch tags updated in DockerHub, and don't only have a vague `latest` without reference to what version of the project you're running. This would be upon a `release` trigger, which the repo doesn't currently do. So, may not be relevant.

Additionally, the name of the docker image isn't a secret, so I'd recommend hard-coding for readability, or how it is in my link with the `docker/metadata-action` action. Example using `docker/metadata-action` for pushing updates on the default branch and `latest` - https://github.com/artis3n/kali-artis3n/blob/main/.github/workflows/cron.yml#L51 .

Reply:
Good points. Unfortunately the project has no releases yet and the name of the owner and project on Docker Hub are currently unknown and must be created by the maintainer.